.github/workflows/verify_docker_image.yml

```diff
@@ -0,0 +1,45 @@
+name: Pull and Verify Docker Image
+
+on:
+  push:
+  registry_package:
+  workflow_call:
+  workflow_dispatch:
+jobs:
+  pull-and-build-image:
+    strategy:
+      fail-fast: false
+      matrix:
+        targets:
+          - amdgpu_target: "gfx942"
+        os: [ubuntu_24_04]
+        runs-on: [linux-mi300-1gpu-ossci-rocm]
```

Comment:
I would have saved this matrix for a follow-up change, to keep this initial PR simpler. Let's try to land small incremental changes so we don't get bogged down by long review threads.

Reply:
Sounds good, so no matrix in this PR and hardcoded links are good then.

Reply:
Yeah let's keep this as simple as possible for the first PR. Direct path to running one test on one runner and gfx target, then we can have follow up PRs add more tests and expand the set of targets / runners.